yt_dlp/extractor/orf.py

```diff
@@ -618,6 +645,8 @@ def _extract_video(self, video_id, display_id):
 
     def _real_extract(self, url):
         video_id, display_id = self._match_valid_url(url).group('id', 'slug')
+        if display_id is None:
+            display_id = video_id
```

Comment:
I don't think we should add a display_id if it is not present
```suggestion
```

Reply:
Yeah, I agree. Especially since it is not really an ID to begin with. This works. https://on.orf.at/video/14210000/pretty-much-anything-you-want